### PR TITLE
Use chrony instead of NTP on latest Fedoras

### PR DESCRIPTION
**Pull request by stbenjam**
Chrony is actually default [since F16](https://fedoraproject.org/wiki/Features/ChronyDefaultNTP).

### CODE REVIEW

File: provisioning_templates/provision/kickstart_default.erb [hunk 1]
```diff
@@ -87,7 +87,12 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
+<% if (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) -%>
```
Comment (lzap) [inline provisioning_templates/provision/kickstart_default.erb]: Can you extract this into helper variable `use_chrony` please? It's three places and it also enables us to provide host param with the same name:

```ruby
use_chrony = (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) || ! host_param('use-ntpd')
```

I want to allow users to use ntpd if they need to.

File: provisioning_templates/provision/kickstart_default.erb [hunk 2]
```diff
@@ -170,9 +179,13 @@ exec < /dev/tty3 > /dev/tty3
 <%= snippet 'kickstart_networking_setup' %>
 <% end -%>
 
-#update local time
-echo "updating system time"
+# Update local time
```
Comment (lzap) [inline provisioning_templates/provision/kickstart_default.erb]: This comment could be easily removed :-)

**Review by lzap — CHANGES_REQUESTED** [review]
Nice, thanks.

**Comment by stbenjam** [comment]
Thanks for the review @lzap, good suggestions.  Updated.

File: provisioning_templates/provision/kickstart_default.erb [hunk 3]
```diff
@@ -87,7 +88,7 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
-<% if (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) -%>
+<% if use_ntp -%>
```
Comment (ekohl) [inline provisioning_templates/provision/kickstart_default.erb:92]: Providing an ntp server is not supported by chrony?

Comment (stbenjam) [reply provisioning_templates/provision/kickstart_default.erb:92]: `--ntpservers` wasn't added to Anaconda until F16 when Chrony was made the default, so this is the opposite case, we don't supply --ntpservers to timezone unless using chrony.

Comment (ekohl) [reply provisioning_templates/provision/kickstart_default.erb:92]: Yes, but if the `use-ntp` param is set to true, those parameters are no longer passed in while they were without it. This line probably needs to stay unchanged.

Comment (stbenjam) [reply provisioning_templates/provision/kickstart_default.erb:92]: I don't follow, we never passed `--ntpservers` to timezone until this PR

Comment (stbenjam) [reply provisioning_templates/provision/kickstart_default.erb:92]: I think you're looking at the wrong diff, i.e. the diff between my two commits, vs. the diff w/ develop.

Comment (ekohl) [reply provisioning_templates/provision/kickstart_default.erb:92]: You are correct.

File: provisioning_templates/provision/kickstart_default.erb [hunk 4]
```diff
@@ -41,7 +42,7 @@ This template accepts the following parameters:
   salt_enabled = host_param('salt_master') ? true : false
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
-%>
+  use_ntp = host_param_true?('use-ntp') || (is_fedora && os_major < 16) || (rhel_compatible && os_major < 8) %>
```
Comment (ekohl) [inline provisioning_templates/provision/kickstart_default.erb]: Could you keep the `%>` on a separate line?

**Comment by stbenjam** [comment]
@ekohl updated

**Review by lzap — APPROVED** [review]

**Comment by lzap** [comment]
I am good, letting @ekohl to merge. Thanks!

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**